RockLib.Compression/IDecompressor.cs

```diff
@@ -12,6 +13,7 @@ public interface IDecompressor
         /// </summary>
         /// <param name="inputStream">The stream to decompress.</param>
         /// <returns>The decompressed byte array.</returns>
+        /// <exception cref="ArgumentNullException"></exception>
```

Comment:
The method on the interface itself doesn't throw `ArgumentNullException`. So...I'm not sure we should be putting this into the comments. It's possible that someone could implement this and not throw that exception.

Reply:
Ah yeah good point.

Reply:
removed.